packages/backend/src/dbt/dbtCliClient.ts

```diff
@@ -140,15 +150,49 @@ export class DbtCliClient implements DbtClient {
                             : DbtManifestVersion.V8,
                     logs: await this._runDbtCommand(dbtExec, ...command),
                 };
-            } catch (e) {
+            } catch (e: unknown) {
                 Sentry.captureException(e, { extra: { dbtExec } });
                 Logger.warn(
                     `Error running ${dbtExec} command "${command.join(
                         ' ',
                     )}": ${e}`,
                 );
+
+                if (!(e instanceof DbtError)) throw e;
+                errorLogs[dbtExec] = e.logs || [];
                 if (dbtExecs[dbtExecs.length - 1] === dbtExec) {
-                    throw e; // Throw last error
+                    throw new DbtError(
+                        `We failed to run "dbt ${command.join(
+                            ' ',
+                        )}" successfully across all version of dbt that Lightdash currently supports. You can see specific errors below for each dbt version we tried to use.`,
```

Comment:
Replace "dbt version we tried to use." with "dbt version we tried to compile with."

Reply:
Is not always during the compile command. This error message is returned independently of the dbt command.

Reply:
ok then leave as is!